src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -30,23 +30,42 @@ export const InviteMemberDialogContent = ({
   groupId,
 }: IInviteMemberDialogContentProps) => {
   const [isLinkButtonClicked, setIsLinkButtonClicked] = useState(false);
+  const [isHash, setHash] = useState('');
 
+  /**
+   * This useEffect is fired when the `isDialogOpen` state is changed(true/false). if true, the setup is fired.
+   * generating the invitation link hash and put it as state `isHash`
+   */
+  useEffect(() => {
+    const getHash = async () => {
+      if (isDialogOpen) {
```

Comment:
```ts
if (!isDialogOpen) return;
```
If you write the first if statement like this, you can avoid nested multiple if statements.